README.md

```diff
@@ -48,10 +48,10 @@ When looking for support, please first search for your question in these venues:
 * [Node.js Website][]
 * [Node.js Help][]
 * [Open or closed issues in the Node.js GitHub organization](https://github.com/issues?utf8=%E2%9C%93&q=sort%3Aupdated-desc+org%3Anodejs+is%3Aissue)
-* [Questions tagged 'node.js' on StackOverflow][]
 
 If you didn't find an answer in one of the venues above, you can:
 
+* View **unofficial** [questions tagged 'node.js' on StackOverflow][]
```

Comment:
Sounds weird. Like there are official question, but these ones are unofficial.
Maybe let's move unofficial to the end? Like

* View  [questions tagged 'node.js' on StackOverflow](https://stackoverflow.com/questions/tagged/node.js) (**unofficial**)

Or

* View  [questions tagged 'node.js' on StackOverflow](https://stackoverflow.com/questions/tagged/node.js) (not supported by Node.js maintainers)

Reply:
@JosephLeon @Trott wdty?

Reply:
Yes, I agree. This is why I wanted to reword the heading so we don't have to find ways to embed *unofficial* in the items. I think the main idea we are trying to convey is that these resources aren't moderated by node.js maintainers.

`If you didn't find an answer in one of the official resources above, you can search these unofficial resources not supported by node.js maintainers:`

Reply:
IMO your suggestion is better than the current version. Let's see what @Trott thinks though.

Reply:
Sounds good to me.

Reply:
(Minor nit: `node.js` -> `Node.js`. Alternatively, you could stop after `unofficial resources` and leave the `which aren't supported by Node.js maintainers` part off. Either way!)

Reply:
I made the changes @Trott asked for but I'm getting a git error today:

`To github.com:JosephLeon/node.git
 ! [rejected]              19412 -> 19412 (non-fast-forward)
error: failed to push some refs to 'git@github.com:JosephLeon/node.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.`

Reply:
@JosephLeon The reason you needed the `-f` is because rebase is destructive and creates new commits on top of master. So when you push, your code doesn't match the server. I think force push is fine in this case because you're the only one working on your branch.